oddbird/oddleventy
# Pagination

## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

![horse's mouth](https://images.unsplash.com/photo-1551884831-bbf3cdc6469e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2081&q=80)

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

https://trello.com/c/mGh7jzeU/83-i-want-pagination-for-resource-lists

## Description
_The commit messages say what you did; this should explain why and/or how._



## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- check out the `/blog/` (resources) page
- pagination links at top & bottom of list
- click through to sub-pages

## Show me
_Provide screenshots/animated gifs/videos if necessary._

![Screenshot_2020-07-06 Resources ](https://user-images.githubusercontent.com/104161/86653181-3b605e80-bfa2-11ea-8c64-8e5e33b87665.jpg)
